utils/PathSanitizingFileCheck

```diff
@@ -72,7 +72,7 @@ constants.""")
         # We are replacing the Unix path separators in the paths passed as
         # arguments with a broader pattern to also allow forward slashes and
         # double escaped slashes in the result that we are checking. Sigh.
-        stdin = re.sub(re.sub(r'/', slashes_re, pattern), replacement, stdin)
+        stdin = re.sub(re.sub(r'\/', slashes_re, re.escape(pattern)), replacement, stdin)
```

Comment:
Could you please upstream this change to `master` and mark with `# SWIFT_ENABLE_TENSORFLOW` in the meantime?